Zend/zend_language_scanner.l

```diff
@@ -1771,28 +1858,35 @@ NEWLINE ("\r"|"\n"|"\r\n")
 }
 
 <ST_IN_SCRIPTING>{TOKENS} {
-	RETURN_TOKEN(yytext[0]);
+	if (yytext[0] == ']' || yytext[0] == ')') {
```

Comment:
I'd suggest splitting this off into a separate lexer rule.

Reply:
OK, working on that.